test-d/jsonify.ts

```diff
@@ -222,6 +222,9 @@ expectType<[string, string]>(tupleJson);
 declare const tupleRestJson: Jsonify<[string, ...Date[]]>;
 expectType<[string, ...string[]]>(tupleRestJson);
 
+declare const mixTupleJson: Jsonify<['1', typeof fn, 2]>;
```

Comment:
Maybe we should just use `null` or an explicit function declaration instead of a variable that's declared somewhere totally different?

```suggestion
declare const mixTupleJson: Jsonify<['1', null, 2]>;
```
```suggestion
declare const mixTupleJson: Jsonify<['1', (_: any) => void, 2]>;
```